test/new-e2e/tests/otel/utils/pipelines_utils.go

```diff
@@ -123,6 +123,89 @@ func TestTraces(s OTelTestSuite, iaParams IAParams) {
 	}
 }
 
+// TestTracesWithSpanReceiverV2 tests that OTLP traces are received through OTel pipelines as expected with updated OTLP span receiver
+func TestTracesWithSpanReceiverV2(s OTelTestSuite, iaParams IAParams) {
```

Comment:
We can remove `iaParams` here and the related code below since we're not testing infra attributes